CardScan.podspec

```diff
@@ -19,13 +19,13 @@ CardScan is a library for scanning credit cards.
 
   s.subspec 'Core' do |core|
     core.source_files = 'CardScan/Classes/**/*'
-    core.resource_bundles = { 'CardScan' => ['CardScan/Assets/*.xcassets', 'CardScan/Assets/*.storyboard', 'CardScan/Assets/*.mlmodelc'] }
+    core.resource_bundles = { 'CardScan' => ['CardScan/Assets/*.xcassets', 'CardScan/Assets/*.storyboard', 'CardScan/Assets/*.mlmodelc', 'CardScan/Assets/Config'] }
```

Comment:
I'm surprised that this file pattern works, but the podspec documentation isn't very clear. You could change it to `CardScan/Assets/Config/*.txt` to be consistent but if it works this is fine also.

Reply:
I thought about that, but I didn't want to limit this to just *.txt in case we add different configurations in the future.